src/themes/uswitch/theme.json

```diff
@@ -233,7 +233,8 @@
     "uswitch-navy": "#141424",
     "light-grey": "#F2F2F6",
     "light-grey-content": "#202020",
-    "border-light-grey": "#E9E9EC"
+    "border-light-grey": "#E9E9EC",
+    "yellow-25": "#FFF4D5"
```

Comment:
do we have all the the other yellows?

Reply:
No, but that's what it was labeled as on figma so I just copied it

Reply:
```json
"primary-yellow-100": "#FFD555",
"primary-yellow-75": "#FFDF80",
"primary-yellow-50": "#FFEAAA",
"primary-yellow-25": "#FFF4D5",

"secondary-yellow-100": "#F7F146",
"secondary-yellow-75": "#F9F574",
"secondary-yellow-50": "#FAF7A2",
"secondary-yellow-25": "#FDFBD0"
```

https://www.figma.com/file/eltmRLsIFOS59UDtRBMxs8/Uswitch-Design-System?node-id=888%3A298